### ISSUE TITLE
Text unexpectedly moves one line up

### ISSUE THREAD

**Issue by KolyaIvankov** [OmniSharp/omnisharp-vscode#4451]
## Issue Description ##

Text in the editor window moves the text along with the cursor one line up each time I start typing, including pressing Del and BackSpace. The behaviour ceases once C# extension is disabled. Disabling other extensions does not mitigate the problem.

## Steps to Reproduce ##

Installing extension for C#, version 1.23.9

## Environment information ##

**VSCode version**: 1.54.3
**C# Extension**: 1.23.9

<details><summary>Dotnet Information</summary>
.NET SDK (reflecting any global.json):
 Version:   5.0.200
 Commit:    70b3e65d53

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.16299
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\5.0.200\

Host (useful for support):
  Version: 5.0.3
  Commit:  c636bbdc8a

.NET SDKs installed:
  2.1.801 [C:\Program Files\dotnet\sdk]
  5.0.200 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.All 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.25 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.25 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 5.0.3 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.25 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 5.0.3 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.12 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 5.0.3 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download
</details>
<details><summary>Visual Studio Code Extensions</summary>

|Extension|Author|Version|
|---|---|---|
|cpptools|ms-vscode|1.2.2|
|csharp|ms-dotnettools|1.23.9|
|godot-tools|geequlim|1.1.2|
|mono-debug|ms-vscode|0.16.2|
|vscode-language-pack-de|MS-CEINTL|1.54.2|
|vscode-mesh-viewer|michead|0.0.10|;

</details>

**Comment by filipw** [comment]
the extension doesn't engage in any line shifting activities - UI rendering is done by VS Code, so this looks like a VS Code problem. I guess one of the VS Code providers (code lens, references provider, completion provider etc) that C# extension register handlers for exhibits weird behavior under some circumstances specific to your machine (zoom level, localization, resolution etc). You can try resetting zoom level to default and if it doesn't help you should open a VS Code issue (https://github.com/microsoft/vscode) instead.

**Comment by KolyaIvankov** [comment]
I have contacted vscode regarding this issue before; it was identified that the issue arises when omnisharp extension is on. They have redirected me here. 
I have restored zoom level to default - even though I have never changed it; the problems continued.
For the sake of consistency, I have checked if similar behaviour kicks in with .cpp files, as I have cpptools extension instlled, as well as xml files - and no, it doesn't, with omnisharp enabled or not.

I will copy your answer in the thread I've opened in vscode.
